### DISCUSSION TITLE
Libre3 Reader and Juggluco

### DISCUSSION THREAD

**Discussion by steve-zero** [maheini/FreeStyle-Libre-3-patch#54] [Q&A]
For reasons of reliabilty, robustness and practicality, we are using the dedicated Libre3 Reader device for our kids.
It would be an interesting backup solution to have Juggluco readily available on my smartphone. But as the dedicated reader does not use LibreView, I do not know what Account ID the device is using to activate the sensor. This would also be important if we want to use a looping insulin pump while sticking to the dedicated reader.

Does anbody know how the Reader Device serial number translates to a virtual Account ID?
Is there any practical means to eavesdrop the sensor activation, in order to determine the ID used by the reader?

Vielen Dank für jede hilreiche Idee.

**Comment by maheini** [comment]
Hi @steve-zero 
Since the activation process most likely will store some kind of "secret key" during the activation online, I don't think there is any chance of re-activating a sensor after the initial start with the reader

**Reply by steve-zero** [reply]
Thanks for the initial thought.
But actually, the sensor activation is an off-line process. Activation with the reader should be similar to the app if no account (and hence no cloud) is used - there, an internal unique ID is used to activate the sensor instead of the account ID. 
See: https://www.juggluco.nl/Juggluco/libre3/nolibreview.html

My hope is that it is derived from the reader's serial number and can be reconstructed.
Sadly, nothing similar to the account ID  or the unique ID is displayed on the reader's information page.
